src/c++/perf_analyzer/genai-pa/tests/test_cli.py

```diff
@@ -91,3 +93,7 @@ def test_arguments_model_not_provided(self):
 
         # Check that the exit was unsuccessful
         assert exc_info.value.code != 0
+
+    def test_execption_on_nonzero_exit(self):
+        with pytest.raises(GenAiPAException) as e:
```

Comment:
Would be good to expand this to look for specific expected error messages when you have specific negative cases you want to test - but LGTM for now.